splashscreen.h

```diff
@@ -0,0 +1,71 @@
+using namespace std;
+
+class Splashscreen {
+    private:
+        sprite background = create_sprite("background.png");
```

Comment:
Just curious as to why the background is a sprite as opposed to a bitmap
```suggestion
        bitmap background = load_bitmap("background", "background.png");
```

Reply:
@Delcari Should probably be a Bitmap, I made this background when I was first beginning Splashkit at the beginning of the project, the UI will have heaps of work, is just things I have thrown in for now so it isn't a white background. The main parts I worked on are getting the basic functionality of the arcade machine working.